<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FtpClientWindow</class>
 <widget class="QDialog" name="FtpClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FtpClientWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="ftpHostLB">
       <property name="text">
        <string>FTP服务器:</string>
       </property>
       <property name="buddy">
        <cstring>ftpHostLE</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ftpHostLE"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="ftpPortLE"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="usernameLB">
       <property name="text">
        <string>用户名:</string>
       </property>
       <property name="buddy">
        <cstring>usernameLE</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="usernameLE"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="passwordLE"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="ftpPortLB">
       <property name="text">
        <string>端口:</string>
       </property>
       <property name="buddy">
        <cstring>passwordLE</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="passwordLB">
       <property name="text">
        <string>密码:</string>
       </property>
       <property name="buddy">
        <cstring>ftpPortLE</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="linkBtn">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upwardBtn">
       <property name="text">
        <string>上级目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadBtn">
       <property name="text">
        <string>下载</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
